### PR TITLE
Add celery configuration

### LINKED ISSUE #197: Configure Celery
Add a dependency on `celery` and configure it.

### PR DESCRIPTION
**Pull request by bartoszbetka**
Resolves #197

### CODE REVIEW

File: concent_api/concent_api/celery.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import os
+from celery import Celery
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'concent_api.settings')
+
+
+app = Celery('concent_api')
+
+# Using a string here means the worker don't have to serialize
```
Comment (cameel) [inline concent_api/concent_api/celery.py]: `Using a string here means that the worker does not have to serialize`

File: concent_api/concent_api/settings/base.py [hunk 2]
```diff
@@ -238,3 +238,6 @@
 # - "Gatekeeper":  An internal helper that validates file transfer tokens.
 # - "admin-panel": Django admin panel that provides access to database content and service statistics.
 # CONCENT_FEATURES = []
+
+# URL format: 'protocol://<user>:<password>@<hostname>:<port>/<virtual host>'
+#CELERY_BROKER_URL = ''
```
Comment (rwrzesien) [inline concent_api/concent_api/settings/base.py]: Isn't pylint complaining about missing space between # and commented code ?

**Comment by rwrzesien** [comment]
@bartoszbetka Can you take a look on the comments and answer and rebase this branch on master ? It would be good to merge this quite soon.

File: concent_api/concent_api/celery.py (hunk 1 above)
Comment (Jakub89) [reply concent_api/concent_api/celery.py]: Done

File: concent_api/concent_api/settings/base.py (hunk 2 above)
Comment (Jakub89) [reply concent_api/concent_api/settings/base.py]: Fixed

**Pull request merged**